### ISSUE TITLE
[CI] FollowerFailOverIT testAddNewReplicasOnFollower failed

### ISSUE THREAD

**Issue by droberts195** [elastic/elasticsearch#38894]
`FollowerFailOverIT.testAddNewReplicasOnFollower` failed in 7.0 in 
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.0+internalClusterTest/530/console

The repro command is:

```
./gradlew :x-pack:plugin:ccr:internalClusterTest \
  -Dtests.seed=19484C0EDDEDE80D \
  -Dtests.class=org.elasticsearch.xpack.ccr.FollowerFailOverIT \
  -Dtests.method="testAddNewReplicasOnFollower" \
  -Dtests.security.manager=true \
  -Dtests.locale=ca-ES \
  -Dtests.timezone=Asia/Ashgabat \
  -Dcompiler.java=11 \
  -Druntime.java=8
```

This didn't reproduce locally for me.

There are 20 assertion errors in the log.  The first one is:

```
10:17:12   2> java.lang.AssertionError: [follower-index] IndexNotFoundException[no such index [follower-index]]
```

Then:

```
10:17:12   2> java.lang.AssertionError: java.lang.RuntimeException: Cluster is already closed
```

Then the one that causes the test failure:

```
10:17:12 ERROR   43.1s J7 | FollowerFailOverIT.testAddNewReplicasOnFollower <<< FAILURES!
10:17:12    > Throwable #1: java.lang.AssertionError: timed out waiting for green state
```

**Comment by droberts195** [comment]
I muted the test for 7.0 in 23716b7b7d815ae610280a0370b1ee4458781fce

**Comment by dnhatn** [comment]
Log: [add_new_replica.zip](https://github.com/elastic/elasticsearch/files/2868004/add_new_replica.zip)

**Comment by dnhatn** [comment]
This was fixed by #38879. I unmuted this test on 7.0. Thanks @droberts195 for reporting this issue.
